### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
I found the README update.  As you suggested @CoryMartin-NOAA github created a branch for my on-line edit.  I've merged this change with develop, so it should be good to go.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # obs-monitor
-Observation Monitoring Pages and Tools
+Observation Monitoring Tools and Pages
+
+## Code Managers:
+Ed Safford @EdwardSafford--NOAA/edward.safford@noaa.gov
```
Comment (CoryMartin-NOAA) [inline README.md]:
```suggestion
Ed Safford @EdwardSafford-NOAA/edward.safford@noaa.gov
```

Comment (CoryMartin-NOAA) [reply README.md]: I'd also just skip adding the emails here, if someone tags us on GitHub we'll get notified anyways

**Review by CoryMartin-NOAA — COMMENTED** [review]
I apparently never clicked submit review... oops see comments now

**Review by ADCollard — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply README.md]: Done and done.

**Review by CoryMartin-NOAA — APPROVED** [review]

**Pull request merged**